kubernetes-manifests/frontend-v2.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend-v2
+spec:
+  selector:
+    matchLabels:
+      app: frontend-v2
+  template:
+    metadata:
+      labels:
+        app: frontend-v2
+    spec:
+      terminationGracePeriodSeconds: 5
+      containers:
+      - name: front
+        image: gcr.io/bank-of-anthos/frontend-v2
+        env:
+        - name: VERSION
+          value: "v0.1.0"
+        - name: LOCAL_ROUTING_NUM
+          value: "123456789"
+        - name: DEFAULT_USERNAME
+          value: "testuser"
+        - name: DEFAULT_PASSWORD
+          value: "password"
```

Comment:
David recenetly submitted a PR to improve how we handle environment variables using configmaps. You should try to follow that pattern: https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/91

Reply:
Updated, thanks!